### DISCUSSION TITLE
Installation tips for beginners?

### DISCUSSION THREAD

**Discussion by pumpknman** [Pitayan/gatsby-theme-pitayan#47] [General]
Hi there. I'm trying to install this Gatsby template. However it seems to be slightly different from other templates. I don't know which files I need to clone in order to use this.

**Comment by daiyanze** [comment]
Hi @pumpknman ! Sorry for the late reply here.
I have to admit that the installation is kinda confusing. And it needs some knowledge about Gatsby.
IF you don't hope to start using this theme from an empty folder, you could clone this folder [`packages/www`](https://github.com/Pitayan/gatsby-theme-pitayan/tree/master/packages/www) and tweak it yourself.

**Reply by pumpknman** [reply]
Thanks @daiyanze. I will try this when I get home today. Thank you for a great theme though.

**Comment by pumpknman** [comment]
Got it done. Even though it's throwing errors when I try to deploy on Netlify, I will try myself!

**Reply by daiyanze** [reply]
I assume this relates to Node version. If so, you could change the Node version in Netlify

refer to this issue https://github.com/Pitayan/gatsby-theme-pitayan/issues/46
